units/meson.build

```diff
@@ -0,0 +1,329 @@
+  ['systemd-journal-catalog-update.service', '',
+   'sysinit.target.wants/'],
+  ['systemd-journal-flush.service',         '',
+   'sysinit.target.wants/'],
+  ['systemd-journal-gatewayd.service',     'ENABLE_REMOTE HAVE_MICROHTTPD'],
+  ['systemd-journal-remote.service',       'ENABLE_REMOTE HAVE_MICROHTTPD'],
+  ['systemd-journal-upload.service',       'ENABLE_REMOTE HAVE_LIBCURL'],
+  ['systemd-journald.service',             '',
+   'sysinit.target.wants/'],
+  ['systemd-kexec.service',                ''],
+  ['systemd-localed.service',              'ENABLE_LOCALED',
+   'dbus-org.freedesktop.locale1.service'],
+  ['systemd-logind.service',               'ENABLE_LOGIND',
+   'multi-user.target.wants/ dbus-org.freedesktop.login1.service'],
+  ['systemd-machine-id-commit.service',    '',
+   'sysinit.target.wants/'],
+  ['systemd-machined.service',             'ENABLE_MACHINED',
+   'dbus-org.freedesktop.machine1.service'],
+  ['systemd-modules-load.service',         'HAVE_KMOD',
+   'sysinit.target.wants/'],
+  ['systemd-networkd-wait-online.service', 'ENABLE_NETWORKD',
+   join_paths(pkgsysconfdir, 'system/network-online.target.wants/')],
+  ['systemd-nspawn@.service',              ''],
+  ['systemd-poweroff.service',             ''],
+  ['systemd-quotacheck.service',           'ENABLE_QUOTACHECK'],
+  ['systemd-random-seed.service',          'ENABLE_RANDOMSEED',
+   'sysinit.target.wants/'],
+  ['systemd-reboot.service',               ''],
+  ['systemd-remount-fs.service',           '',
+   'local-fs.target.wants/'],
+  ['systemd-rfkill.service',               'ENABLE_RFKILL'],
+  ['systemd-suspend.service',              ''],
+  ['systemd-sysctl.service',               '',
+   'sysinit.target.wants/'],
+  ['systemd-sysusers.service',             'ENABLE_SYSUSERS',
+   'sysinit.target.wants/'],
+  ['systemd-timedated.service',            'ENABLE_TIMEDATED',
+   'dbus-org.freedesktop.timedate1.service'],
+  ['systemd-timesyncd.service',            'ENABLE_TIMESYNCD',
+   join_paths(pkgsysconfdir, 'system/sysinit.target.wants/')],
+  ['systemd-tmpfiles-clean.service',       'ENABLE_TMPFILES'],
+  ['systemd-tmpfiles-setup-dev.service',   'ENABLE_TMPFILES',
+   'sysinit.target.wants/'],
+  ['systemd-tmpfiles-setup.service',       'ENABLE_TMPFILES',
+   'sysinit.target.wants/'],
+  ['systemd-udev-settle.service',          ''],
+  ['systemd-udev-trigger.service',         '',
+   'sysinit.target.wants/'],
+  ['systemd-udevd.service',                '',
+   'sysinit.target.wants/'],
+  ['systemd-update-done.service',          '',
+   'sysinit.target.wants/'],
+  ['systemd-update-utmp-runlevel.service', 'HAVE_UTMP HAVE_SYSV_COMPAT',
+   'multi-user.target.wants/ graphical.target.wants/ rescue.target.wants/'],
+  ['systemd-update-utmp.service',          'HAVE_UTMP',
+   'sysinit.target.wants/'],
+  ['systemd-user-sessions.service',        '',
+   'multi-user.target.wants/'],
+  ['systemd-vconsole-setup.service',       'ENABLE_VCONSOLE'],
```

Comment:
I did build with ```-Dvconsole=false```, yet systemd-vconsole.service was installed at /lib/systemd/system/systemd-vconsole.service. Seems to be a general problem.